pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/util/SchemaUtil.java

```diff
@@ -32,12 +32,16 @@
 
 public class SchemaUtil {
 
+    public static boolean DEFAULT_JSR310_CONVERSION_ENABLED = false;
+
     public static boolean getJsr310ConversionEnabledFromSchemaInfo(SchemaInfo schemaInfo) {
-        if (schemaInfo != null) {
-            return Boolean.parseBoolean(schemaInfo.getProperties()
-                    .getOrDefault(SchemaDefinitionBuilderImpl.JSR310_CONVERSION_ENABLED, "false"));
```

Comment:
Hi @adrianiacobghiula 

Here uses `getOrDefault`, which should already provide the ability to override. Do you have a test to reproduce this issue?

Reply:
Hi @codelipenghui,
Thank you for having a look on my PR. the getOrDefault is not enough. I want to be able to override what the schema defines if the jsr310 conversion is or not enabled. The schema can come from an old producer, from a golang producer. In my code I want to be able to decide if my client is able to do the jsr310 conversion and not the schema.  I added the tests :)